lib/src/util/console_log_utils.dart

```diff
@@ -152,3 +184,30 @@ class ConsoleConfig {
   /// Captures calls to `console.log`, `console.warn`, and `console.error`.
   static const ConsoleConfig all = ConsoleConfig._('all');
 }
+
+dynamic _jsFunctionApply(dynamic jsFunction, List<dynamic> args) {
+  return callMethod(jsFunction, 'apply', [null, jsify(args)]);
```

Comment:
#nit Is there a reason we don't pass in a `this` arg (from `allowInteropCaptureThis`) instead of `null` here, like we were doing before?

It looks like it works without it in Chrome, but it seems like it'd be nice to have to be safe.

Reply:
That was what i did first but when i tried that it had issues, i don't recall exactly what it was but then i tried with null instead and it was passing so i went with it. lol

Reply:
and of course i try it again and it works...